.github/workflows/create-release-pr.yml

```diff
@@ -0,0 +1,93 @@
+name: Create release PR
+
+on: workflow_dispatch
+
+jobs:
+  create-release-pr:
+    runs-on: ubuntu-20.04
+    defaults:
+      run:
+        working-directory: ./main
+    steps:
+      - name: Check out the woocommerce-subscriptions-core repository
+        uses: actions/checkout@v3
+        with:
+          ref: trunk
+          path: main
+
+      - name: Check out the woorelease repository
+        uses: actions/checkout@v3
+        with:
+          repository: woocommerce/woorelease
+          ref: 2.2.0
+          # This token uses the "repo" scope to check out the private Woorelease repo.
+          # This personal access token is owned by the botwoo account - https://github.com/botwoo
+          # Note: GITHUB_TOKEN does not work here.
+          token: ${{ secrets.GH_REPO_PAT }}
+          path: woorelease
+
+      - name: Enable composer dependencies caching
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/composer/
+          key: ${{ runner.os }}-composer-${{ hashFiles('woorelease/composer.lock') }}
+
+      - name: Enable npm dependencies caching
+        uses: actions/cache@v2
+        with:
+          path: ~/.npm/
+          key: ${{ runner.os }}-npm-${{ hashFiles('main/package-lock.json') }}
+
+      - name: Setup PHP with composer
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '7.4'
+          tools: composer
+          coverage: none
+
+      - name: Install Woorelease composer dependencies
+        run: |
+          cd $GITHUB_WORKSPACE/woorelease
+          composer install --prefer-dist --no-dev
+
+      - name: Get version
+        id: get_version
+        run: |
+          VERSION=$(sed -n '/^= /s/^.*[^0-9]\([0-9]*\.[0-9]*\.[0-9]*\).*$/\1/p' changelog.txt | head -1)
+
+          echo "::set-output name=VERSION::$(echo $VERSION)"
+
+      - name: Create the release branch
+        env:
+          VERSION: ${{ steps.get_version.outputs.VERSION }}
+        run: |
+          git checkout -b release/$VERSION
+
+      - name: Update version
+        env:
+          VERSION: ${{ steps.get_version.outputs.VERSION }}
+        run: |
+          php bin/update-version.php $VERSION
```

Comment:
Hey @brucealdridge, I'm not sure I understand your request.

I'll provide some extra context below to ensure we're on the same page.

Let's use your example, i.e. current version is `1.8.0` and the upcoming version is `1.9.0`.

The `changelog.txt` file will contain the following header:

```
= 1.9.0 - 2022-xx-xx =
```

In the "Get version" step we parse the `changelog.txt` file and grab the upcoming version (`1.9.0` in this example). We then set that version as an output parameter so we can access it in subsequent steps.

We use that version in the "Create the release branch" step to create the release branch (e.g. `release/1.9.0`)

We also use it in the "Update version" step to bump the version strings across the codebase (`package.json`, `woocommerce-subscriptions-core.php`, etc.) 

These steps do not reference the current version (i.e. `1.8.0`). The steps only ever need to reference the upcoming version (i.e. `1.9.0`).

Hopefully that makes sense 😅

Reply:
Thanks, that makes sense. 

So we need to make sure that we have a version number included in the changelog rather than a placeholder like
```
= 1.x.x - 20xx-xx-xx =
```
I misread the code, or assumed that woorelease was bumping the version. But rather it is just updating the version in the various places to the version that is passed in (and already in the changelog.)

Reply:
FYI: https://github.com/Automattic/woocommerce-subscriptions-core/commit/c44ec3ed415bb54c96e47e4ffe8b48605cba0152
It looks like it's happened a couple of times. Will just need to make sure we commit an actual version number before running the action. 
If this happens, the action will either fail or produce a `Version x.x.x` PR that we can just close so not a problem.